src/test/java/com/vite/service/ViteHandlerTest.java

```diff
@@ -1,16 +1,19 @@
 package com.vite.service;
 
 import org.junit.Test;
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
 
 
 public class ViteHandlerTest {
     /**
      * Tests the {@link ViteHandler#ping()} function
-     * TODO add assertion about log statement, or better yet make ping return a String
      */
     @Test
     public void testPing() {
         ViteHandler viteService = new ViteHandler();
-        viteService.ping();
+        String response = viteService.ping();
+        String expected = "Server Response to ping at ";
+        assertThat(response.contains(expected), is(true));
```

Comment:
Matcher style reeds like english bro?

Assert that response contains expected is true. English.